Examples/Algorithms/Fatras/src/FatrasSimulation.cpp

```diff
@@ -120,15 +120,20 @@ struct FatrasSimulationT final : ActsExamples::detail::FatrasSimulation {
 
   FatrasSimulationT(const ActsExamples::FatrasSimulation::Config &cfg,
                     Acts::Logging::Level lvl)
-      : simulation(
-            ChargedSimulation(
-                ChargedPropagator(ChargedStepper(cfg.magneticField),
-                                  Acts::Navigator{{cfg.trackingGeometry}}),
-                Acts::getDefaultLogger("Simulation", lvl)),
-            NeutralSimulation(
-                NeutralPropagator(NeutralStepper(),
-                                  Acts::Navigator{{cfg.trackingGeometry}}),
-                Acts::getDefaultLogger("Simulation", lvl))) {
+      : simulation(ChargedSimulation(
+                       ChargedPropagator(
+                           ChargedStepper(cfg.magneticField),
+                           Acts::Navigator{{cfg.trackingGeometry},
+                                           Acts::getDefaultLogger("nav", lvl)},
+                           Acts::getDefaultLogger("prop", lvl)),
+                       Acts::getDefaultLogger("Simulation", lvl)),
+                   NeutralSimulation(
+                       NeutralPropagator(
+                           NeutralStepper(),
+                           Acts::Navigator{{cfg.trackingGeometry},
+                                           Acts::getDefaultLogger("nav", lvl)},
+                           Acts::getDefaultLogger("prop", lvl)),
+                       Acts::getDefaultLogger("Simulation", lvl))) {
```

Comment:
Seems like an orthogonal change. Other than that, we could start with a single logger and then clone them with suffix?

Reply:
right I just did not have the space for a temporary object but I think I will drop the change from here